test/system/src/main/java/io/pravega/test/system/framework/services/docker/ZookeeperDockerService.java

```diff
@@ -58,7 +58,7 @@ private ServiceSpec setServiceSpec() {
 
         final TaskSpec taskSpec = TaskSpec
                 .builder()
-                .containerSpec(ContainerSpec.builder().image(ZK_IMAGE)
+                .containerSpec(ContainerSpec.builder().image(IMAGE_PATH + IMAGE_PREFIX + "/zookeeper:" + ZK_VERSION)
```

Comment:
we should make the image name configurable.

Reply:
Made the changes